src/plugins/data/server/search/es_search/es_search_strategy.ts

```diff
@@ -49,6 +49,9 @@ export const esSearchStrategyProvider = (
         const response = shimHitsTotal(body, options);
         return toKibanaSearchResponse(response);
       } catch (e) {
+        if (e.body?.error) {
```

Comment:
are we sure we want to log every error ? cc @lizozom

Reply:
I am going to revert this. It seems that it also logs warning messages and makes them look like an error.

The reason I added this originally is that when there is a 400-status response from Elasticsearch, the error result includes an explanation of why the request failed. Without logging the error, the explanation is completely lost.

We should find a way to log the explanation of the error _consistently_, as sometimes it is something the user can fix.

Reply:
I was going to file an issue about this, but I realized that since the `KbnServerError` is thrown, the consumer can catch it themselves and add the extra logging if there is any needed.